nats/src/main/java/io/micronaut/nats/reactive/ReactivePublisher.java

```diff
@@ -24,25 +24,25 @@
  */
 public interface ReactivePublisher {
 
+
     /**
      * Publish the message with the provided arguments and return
      * a reactive type that completes successfully when the message
      * is published.
      *
-     * @param publishState The publishing data
+     * @param message The message to publish
      * @return the publisher
      */
-    Publisher<Void> publish(PublishState publishState);
+    Publisher<Void> publish(Message message);
```

Comment:
This would be a breaking change, so need to deprecate old and add new methods

Reply:
See further discussion.